profiles/akhila_pathapati.md

```diff
@@ -0,0 +1,14 @@
+# First and Last Name
```

Comment:
Add your name inside the hashtag like `# Akhila Pathapati`
- Also remove the second like which has your name in it
- You can just push a new commit to override the changes